sklearn/metrics/_plot/roc_curve.py

```diff
@@ -665,8 +665,8 @@ def from_cv_results(
 
         pos_label : int, float, bool or str, default=None
             The class considered as the positive class when computing the ROC AUC
-            metrics. By default, `estimators.classes_[1]` is considered
-            as the positive class.
+            metrics. By default, `estimator.classes_[-1]` (using `estimator` from
```

Comment:
Technically we do use `estimator.classes_[-1]` but I can switch back to `[1]` if you think `-1` is confusing.

Reply:
I think you mean `[1]`, I had a quick search and saw that this seems to be what we have elsewhere?
Yes can change back to this. I just don't like it because it is technically not what happens - if we just happen to only have 1 class, `[1]` would give an error, whereas `[-1]` (what we actually do) would not - though a different error/warning would probably arise due to there only being one class.

But much of a muchness. `[1]` is easier for users to understand.

Reply:
I will change to [1]